src/frontend/components/DashboardTimedTextManager/index.spec.tsx

```diff
@@ -80,7 +80,7 @@ describe('<DashboardTimedTextManager />', () => {
       ),
     );
 
-    await wait();
+    await waitFor(() => {});
```

Comment:
I think the purpose of the move from `wait` to `waitFor` was to avoid the logic of "wait for the next process tick" which is considered brittle by the builders of `testing-library` [Kent talks about it here](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#passing-an-empty-callback-to-waitfor).

For async based waits, I have not had any issues with the `Deferred` approach I suggest above.

For other needs, Kent suggests using `find` [just above the previous one](https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-waitfor-to-wait-for-elements-that-can-be-queried-with-find) which is `getBy` packaged with `waitFor` in one call.

Reply:
Wow it looks like I didn't read last improvement in this lib since a while...